functionary/builder/utils.py

```diff
@@ -191,16 +190,15 @@ def build_package(build_id: UUID):
             build.package = package
             build.save()
 
-    try:
-        # Need to validate the potentially new function schema, but
-        # don't save it until the build has finished.
-        db_functions = _create_functions_from_definition(
-            package_definition.get("functions"), package
-        )
-        _deactivate_functions(package_definition.get("functions"), package)
-        _extract_package_contents(package_contents, workdir)
-        _load_dockerfile_template(dockerfile, workdir)
+    # Need to validate the potentially new function schema, but
+    # don't save it until the build has finished.
+    functions, function_parameters = _create_functions_from_definition(
```

Comment:
This got bumped outside of the try block since:

* The exceptions being handled didn't seem like they could be raised by the lines here.
* The linter was rightfully complaining later on that technically `functions` could be unset.